src/azure-cli/setup.py

```diff
@@ -74,7 +74,7 @@
     'azure-mgmt-compute~=12.0',
     'azure-mgmt-consumption~=2.0',
     'azure-mgmt-containerinstance~=1.4',
-    'azure-mgmt-containerregistry==3.0.0rc12',
+    'azure-mgmt-containerregistry~=3.0.0rc13',
```

Comment:
why use `~=`, could  you pls pin exact version again? thanks.

Reply:
@yungezz I did this because other dependencies aren't pinned to an exact version. Do you want me to do something different?

Reply:
I remembered we had the problem of breaking change in patch version from 9.0 -> 11.0.  that's the reason we pinned the exact version at that time.  What's your thinking on this.

Reply:
@adewaleo let us just use `==` to pin it

Reply:
@yungezz @yugangw-msft sounds good! Will pin.